dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -29,10 +29,10 @@ canonicalHostName=Use canonical host name
 canonicalHostName.desc= Controls whether you want to use the canonical host name.
 
 krb5Config=The Kerberos configuration file with full path
-krb5Config.desc=Specifies the fully qualified Kerberos configuration path and name. Standard variable substitutions, such as ${server.config.dir}, can be used when specifying the directory path.
+krb5Config.desc=Deprecated: use configFile attribute on the <kerberos> element instead. Specifies the fully qualified Kerberos configuration path and name. Standard variable substitutions, such as ${server.config.dir}, can be used when specifying the directory path.
```

Comment:
**change**
Standard variable substitutions, such as ${server.config.dir}, can be used when specifying the directory path.
**to**
Standard variable substitutions, such as ${server.config.dir}, can be used when you specify the directory path.